File: app/src/main/java/com/checkin/app/checkin/Auth/AuthenticatorService.java
package com.checkin.app.checkin.Auth;

import android.app.Service;
import android.content.Intent;
import android.os.IBinder;
import androidx.annotation.Nullable;

public class AuthenticatorService extends Service {
    private AccountAuthenticator mAuthenticator;

    @Override
    public void onCreate() {
        mAuthenticator = new AccountAuthenticator(this);
    }

    @Nullable
    @Override
    public IBinder onBind(Intent intent) {
        return mAuthenticator.getIBinder();
    }
}

File: app/src/main/java/com/checkin/app/checkin/auth/services/AuthenticatorService.kt
package com.checkin.app.checkin.auth.services

import android.app.Service
import android.content.Intent
import android.os.IBinder

class AuthenticatorService : Service() {
    private lateinit var mAuthenticator: AccountAuthenticator

    override fun onCreate() {
        mAuthenticator = AccountAuthenticator(this)
    }

    override fun onBind(intent: Intent): IBinder? {
        return mAuthenticator.iBinder
    }
}